pyproject.toml

```diff
@@ -78,6 +78,21 @@ requires = ["poetry>=0.12"]
 build-backend = "poetry.masonry.api"
 
 
+[tool.pytest]
+filterwarnings = ["error::UserWarning"]
```

Comment:
I realize this is just moving an existing setting around, but I'm curious what this is actually doing. Maybe causing `UserWarning`s to be elevated to errors?

Reply:
It's based on [warnings.simplefilter](https://docs.python.org/3/library/warnings.html#warnings.simplefilter) it seems. Your speculation was spot on :)